### PR TITLE
Bug 1914926:  Upgrade Protractor to pick up webdriver-manager fix for…

### PR DESCRIPTION
**Pull request by rhamilto**
… MacOS

See https://github.com/angular/webdriver-manager/issues/476

### CODE REVIEW

File: frontend/yarn.lock [hunk 1]
```diff
@@ -14943,9 +14959,9 @@ protractor-jasmine2-screenshot-reporter@0.5.x:
     uuid "^2.0.0"
 
 protractor@5.4.x:
-  version "5.4.2"
-  resolved "https://registry.yarnpkg.com/protractor/-/protractor-5.4.2.tgz#329efe37f48b2141ab9467799be2d4d12eb48c13"
-  integrity sha512-zlIj64Cr6IOWP7RwxVeD8O4UskLYPoyIcg0HboWJL9T79F1F0VWtKkGTr/9GN6BKL+/Q/GmM7C9kFVCfDbP5sA==
+  version "5.4.4"
```
Comment (spadgett) [inline frontend/yarn.lock:14962]: Update protractor to `^5.4.4` in package.json as well?

Comment (rhamilto) [reply frontend/yarn.lock:14962]: Certainly won't hurt.  Updated.

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**